tests/data_files/test-ca2.ku-crt_crl.crt.openssl.v3_ext

```diff
@@ -0,0 +1,4 @@
+basicConstraints = CA:true
+subjectKeyIdentifier=none
```

Comment:
Maybe there is some config that I'm missing, but is `none` accepted? When I try to generate the certificate with `make`, openssl throws the following error.
```
Error Loading extension section default
139977640584896:error:22071071:X509 V3 routines:string_to_hex:illegal hex digit:v3_utl.c:498:
139977640584896:error:22098080:X509 V3 routines:X509V3_EXT_nconf:error in extension:v3_conf.c:95:name=subjectKeyIdentifier, value=none
```

Reply:
I can reproduce this failure with openssl 1.1.1f but not with 3.0.2.

I think the makefile already has commands that require openssl ≥3 and others that don't work with openssl ≥3, so resolving this is out of scope here.